src/UglyToad.PdfPig/Tokenization/Scanner/PdfTokenScanner.cs

```diff
@@ -256,35 +256,23 @@ private bool TryReadStream(long startStreamTokenOffset, bool getLength, out Stre
 
             // Verify again that we start with "stream"
             var hasStartStreamToken = ReadStreamTokenStart(inputBytes, startStreamTokenOffset);
-
             if (!hasStartStreamToken)
             {
                 return false;
             }
 
             // From the specification: The stream operator should be followed by \r\n or \n, not just \r.
-            if (!inputBytes.MoveNext())
-            {
-                return false;
-            }
-
-            // While the specification demands a \n we have seen files with \r only in the wild.
-            var hadWhiteSpace = false;
-            if (inputBytes.CurrentByte == '\r')
+            // While the specification demands a \n we have seen files with `garbage` before the actual data
+            do
             {
-                hadWhiteSpace = true;
-                inputBytes.MoveNext();
-            }
-
-            if (inputBytes.CurrentByte != '\n')
-            {
-                if (!hadWhiteSpace)
+                if (!inputBytes.MoveNext())
                 {
                     return false;
                 }
+            } while (char.IsWhiteSpace((char)inputBytes.CurrentByte));
```

Comment:
I think we specifically want to stop on the first `\n` here to be compliant with the spec since the data could start with any whitespace, for example where 0x20 is whitespace:

```
stream0x200x20\r\n
0x205ay36
```